src/app/shared/version-picker/version-picker.html

```diff
@@ -3,7 +3,7 @@
   <mat-icon>arrow_drop_down</mat-icon>
 </button>
 <mat-menu #versionPicker="matMenu">
-  <button mat-menu-item *ngFor="let version of docVersions | async"
+  <button mat-menu-item *ngFor="let version of docVersions"
```

Comment:
AsyncPipe removed due to the interaction of types defined for NgForOf and AsyncPipe (tracked in https://github.com/angular/angular/pull/31371).

Alternatives were to type it as `Observable<any>` or disable one of `strictNullChecks`, `enableIvy`, or `fullTemplateTypeCheck`.
Reproduction and more details at: https://github.com/Splaktar/ngfor-asyncpipe-ivy.

Reply:
It looks like this workaround may work:
https://github.com/angular/angular/pull/32980/files?file-filters%5B%5D=.html#diff-986a7d69debab319598e3a2e92ea615dR24

Reply:
and it does. Will update in a follow up PR.